lib/solr_wrapper/instance.rb

```diff
@@ -101,37 +101,41 @@ def restart
     end
 
     ##
-    # Check the status of a managed Solr service
-    def status
-      return true unless config.managed?
+    # Is Solr running?
+    def started?
+      status_info =~ /running on port #{port}/ && true || false
```

Comment:
Actually, I'm going to push up another commit to replace this with an `String#include?` call instead, which seems cleaner and could be faster although that's not important here.
- http://www.mervine.net/ruby-methods-vs-regex